frontend/src/components/ImplementationReportView/ImplementationReportView.tsx

```diff
@@ -0,0 +1,126 @@
+import { Card } from "react-bootstrap";
+import { useLoaderData, Link } from "react-router-dom";
+import { Table, Container } from "react-bootstrap";
+import { ReportData, ImplementationMetadata } from "../../data/parseReportData";
+// @ts-ignore
+import LoadingAnimation from "../LoadingAnimation";
+import DialectCompliance from "./DialectCompliance";
+import { mapLanguage } from "../../data/mapLanguage";
+
+export const ImplementationReportView = () => {
+  const loaderData = useLoaderData() as { [key: string]: ReportData };
+
+  const pathSegments = window.location.href.split("/");
+  let implementationName = pathSegments[pathSegments.length - 1];
+
+  let implementationDetail: ImplementationMetadata = {
+    dialects: [],
+    homepage: "",
+    name: "",
+    version: "",
+    language: "",
+    issues: "",
+    image: "",
+  };
+
+  let allImplementations: { [key: string]: ImplementationMetadata } = {};
```

Comment:
This is my biggest concern I guess - you transform data in this component and also in `DialectCompliance`. Ideally, all this work could be done way up in a router loader.

Also please have a look at `calculateTotals` function in `SummaryTable.tsx` (probably should've put it somewhere else :P). I think if in a router loader we could get the data to look like something like this:
```ts
{
  [key: string]: Totals
}
```
plus the implementation metadata, then it would be all we need?

Reply:
Done!

Now, the loader data looks like:
```js
{
    "name": "...",
    "homepage": "...",
    "issues": "...",
    ...
    "results": {
        "drafts": Totals
    }
}
```